src/os.toit

```diff
@@ -16,6 +16,9 @@ class EnvironmentVariableMap:
     if not result: throw "ENV NOT FOUND"
     return result
 
+  operator []= key/string value/string -> none:
```

Comment:
Maybe add a comment.
My interpretation is that it
- modifies the environment so that future `get` calls read the new value.
- all new subprocesses get the new value.
- no other effect